### PR TITLE
Prepare for 0 7 0

### LINKED ISSUE #13: Package aggregation juttle as adapter module
Now that juttle 0.7.0 supports adapter modules, start packaging the aggregation juttle code in the adapter so it can be accessed as a module.

### PR DESCRIPTION
**Pull request by mstemm**
Make changes necessary to support juttle 0.7.0. This is mostly just switching to adapter modules, but the juttle for the module also required minor changes for nested point support.

There's a current test failure I don't understand. Two test cases related to unary expressions are passing now when they should be failing. @dmajda can you take a look?

This fixes #13.

### CODE REVIEW

**Comment by dmajda** [comment]
> There's a current test failure I don't understand. Two test cases related to unary expressions are passing now when they should be failing. @dmajda can you take a look?

This is a (stupid) bug, see https://github.com/juttle/juttle/issues/650. I’m going to fix it right away.

**Comment by mstemm** [comment]
I think there's still a problem. The filtering expression 'product = ! "EC2"' is being parsed as:

```{ type: 'BinaryExpression',
  location:
   { filename: null,
     start: { offset: 0, line: 1, column: 1 },
     end: { offset: 17, line: 1, column: 18 } },
  operator: '==',
  left:
   { type: 'Field',
     location: { filename: null, start: [Object], end: [Object] },
     name: 'product',
     d: false },
  right:
   { type: 'Field',
     location: { filename: null, start: [Object], end: [Object] },
     name: 'EC2',
     d: false },
  d: false }```

I don't see where the ! operator is being applied to the EC2.

**Comment by dmajda** [comment]
I can’t see anything wrong in the current code and in `juttle` master everything seems to work as it should. Maybe you used `juttle` version without the fix or didn’t regenerate the parser (`gulp peg`)?

**Comment by mstemm** [comment]
Yeah, that was it. Running gulp peg fixed it.

**Comment by mstemm** [comment]
@VladVega @go-oleg @demmer @davidvgalbraith

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # Change Log
 This file documents all notable changes to juttle-aws-adapter. The release numbering uses [semantic versioning](http://semver.org).
 
+## 0.3.0
+Released 2016-03-23
+
+### Minor Changes
+- Update to reflect changes in juttle 0.7.0. This release is compatible with adapter version 0.7.0. [[#12](https://github.com/juttle/juttle-aws-adapter/pull/12)].
```
Comment (demmer) [inline CHANGELOG.md]: How about we keep it simple and just say "Updated to be compatible with Juttle 0.7.x".

**Comment by demmer** [comment]
+1

**Pull request merged**